nwbinspector/nwbinspector.py

```diff
@@ -66,6 +66,20 @@ def copy_function(function):
     return copied_function
 
 
+def load_config(filepath_or_keyword: PathType) -> dict:
```

Comment:
you are requiring filepath_or_keyword to be PathType?

Reply:
I don't see what this refactor has to do with skipping validation

Reply:
`PathType=Union[Path, str]`, which is technically what this argument takes, though the type name may be a bit misleading in that respect, since the `str` part could be either a string path or a string keyword.

From PR description: Add small helper for loading internal configs; this was previously only a part of the CLI, this just makes the code usable importable from the library.

It's easier to setup both these changes so the DANDI PR can point to a single branch for the fixes